### DISCUSSION TITLE
I raised this the first one before…

### DISCUSSION THREAD

**Discussion by fergoid** [ekristen/azure-nuke#50] [Ideas]
If I configure a resource group to be filtered then I would like a way to filter out all of its child resources in a single filter / shorthand of some sort.

Being able to filter resources based on tags would be great.  In fact that is one of my primary use cases.  If your RG is not tagged with a known value then it will be removed along with its child resources.  I’ve used tag filters quite successfully on AWS nuke to control what is removed , albeit not with your enhanced version.

**Comment by ekristen** [comment]
hey @fergoid somehow this went missed. I apologize. I'm getting ready to drop a new beta of a major overhaul to include global filters and much more. Filtering on tags is supported as long as those tags are being added to the resource. If there are specific resources you are not seeing this on, please let me know.
